vme/src/act_wizard.cpp

```diff
@@ -204,7 +204,7 @@ void do_crash(unit_data *ch, char *argument, const command_info *cmd)
 
 void do_execute(unit_data *ch, char *argument, const command_info *cmd)
 {
-    argument = skip_spaces(argument);
+    argument = (char *) skip_spaces(argument);
```

Comment:
No technical reason but I'd use c++ casts, because they are so ugly they are easy to grep for and makes me want to get rid of them sooner

Reply:
I'd write them if I could remember them :-D

Reply:
:-) They are static_cast, reinterpret_cast, const_cast, dynamic_cast

When you do a c-style cast it actually tries c++ casts under the hood in this order
1. const_cast
2. static_cast
3. static_cast and a const_cast (to remove const)
4. reinterpret_cast
5 reinterpret_cast and a const_cast (to remove const)